lib/load-rules.js

```diff
@@ -29,6 +31,11 @@ module.exports = function(rulesDir, cwd) {
         rulesDir = path.resolve(cwd, rulesDir);
     }
 
+    // cache will help performance as IO operation are expensive
+    if (rulesDirCache[rulesDir]) {
```

Comment:
This cache is only when we do a IO operation to scan a directory for rule files. If someone uses 2 instances of linter than we dont want to perform the IO operation 2 times for the same directory.
By default, core rules directory is always included.